### PR TITLE
feat: add freemail app

### PR DESCRIPTION
**Pull request by eNkru**
add the freemail app

### CODE REVIEW

**Comment by eNkru** [comment]
> I don't think this app submission can be accepted as-is -- please see https://github.com/electron/apps/blob/master/contributing.md#company-logos-and-names.
> 
> The name listed here is 'Freelook' but https://github.com/eNkru/electron-outlook flatly says the name is "Electron Outlook" and the one-sentence description is "The Outlook dekstop client."
> 
> If you'd like this to be listed in the Electron apps page, please consider using the 'Freelook' title in earnest and using phrasing more like "Freelook, an Electron-based client for Microsoft Outlook".

I've updated my project and the pull request according the suggestion. Please review it again. Thanks.

File: apps/freelook/freelook.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Freelook
+description: 'Freelook, an Electron-based client for Microsoft Outlook.'
```
Comment (ckerr) [inline apps/freelook/freelook.yml]: You can make CI happy by not starting the description with the appname :smiley: 

```text

  14105 passing (3s)
  1 failing
  1) human-submitted app data
       freelook
         freelook.yml
           should not start description with app name:
      AssertionError: expected 0 to not equal 0
      + expected - actual
```

Comment (eNkru) [reply apps/freelook/freelook.yml]: > You can make CI happy by not starting the description with the appname 

It's been fixed and all green :)

**Comment by groundwater** [comment]
FYI: https://github.com/electron/apps/pull/1820

**Pull request merged**